src/com/amazon/ion/impl/lite/IonStructLite.java

```diff
@@ -629,38 +584,17 @@ public void put(String fieldName, IonValue value)
     }
 
     @Override
-    public ListIterator<IonValue> listIterator(int index) {
-        return new SequenceContentIterator(index, isReadOnly()) {
```

Comment:
Removing this override resulted in a speedup of up to 13% depending on the dataset.